### ISSUE TITLE
test for presence of serverlist should be after sourcing podgetrc

### ISSUE THREAD

**Issue by Corin-EU** [dvehrs/podget#38]
The configuration file podgetrc allows the setting of a custom name for the serverlist file with the CONFIG_SERVERLIST variable.

However, the podget shell script tests if the default named serverlist file is present in the ${DIR_CONFIG} directory before the podgetrc configuration is sourced.

Therefore if an alternative name to "serverlist" is specified in the podgetrc, a default serverlist file is needlessly created in ${DIR_CONFIG}.  This does not happen if the alternative name is specified with the command line --serverlist parameter.

The testing for the presence of the the serverlist file by the name, if specified in podgetrc, should be after the podgetrc is sourced.

**Comment by dvehrs** [comment]
Interesting find.  

I think I might have a fix for it.  I divided the existing configuration tests and creation into four parts.  Podgetrc is now read after the first three and the existence of the serverlist is tested in the fourth part.

I also had to move around the setting of the CONFIG_SERVERLIST to match the setting given on the command line if that option is used so it wasn't replaced with the setting from podgetrc.  

I believe I've got everything in the right order now.  I've uploaded my attempted fixes to the DEV branch.  Feel free to test it and let me know if it works for you.

Thanks for the report!

Dave
